### PR TITLE
test: added unit tests for invalid dump and version

### PR DESCRIPTION
**Pull request by danielskinstad**
Ticket: MEN-6897

### CODE REVIEW

File: conf/version_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	Version = "1.0"
+	v := VersionString()
+	assert.Equal(t, v, Version)
+}
+
+func TestMenderShowVersion(t *testing.T) {
+	Version = "v1.0"
+	v := ShowVersion()
+	assert.True(t, strings.Contains(v, "v1.0"))
+}
```
Comment (lluiscampos) [inline conf/version_test.go]: These tests have very little value: we are setting a variable and then checking that it is returned.

Can we instead move it into the `cli_test.go` package and check that `mender-snapshot --version` prints the right thing? You can look at [mender-artifact cli tests](https://github.com/mendersoftware/mender-artifact/blob/master/cli/cli_test.go) for some inspiration

File: cli/cli_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	defer tty.Close()
+
+	os.Stdout = tty
+
+	args := []string{"snapshot", "dump"}
+	err = SetupCLI(args)
+	if err == nil {
+		log.Error("Expected error")
+	}
+	assert.ErrorContains(t, err, "Refusing to write to terminal")
```
Comment (lluiscampos) [inline cli/cli_test.go:44]: Nice test :sparkles:

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Thank you. An small request to move the version tests to have more value. See below.

**Review by lluiscampos — APPROVED** [review]

**Comment by danielskinstad** [comment]
@mender-test-bot start pipeline

**Pull request merged**